test/manifold_test.cpp

```diff
@@ -784,3 +784,56 @@ TEST(Manifold, EmptyHull) {
       {0, 0, 0}, {1, 0, 0}, {0, 1, 0}, {1, 1, 0}};
   EXPECT_TRUE(Manifold::Hull(coplanar).IsEmpty());
 }
+
+TEST(Manifold, ConvexConvexMinkowski) {
```

Comment:
Let's move these to boolean_test, since that's what they're exercising. We should probably break these files down more someday...